docs/project/development-guidelines.md

```diff
@@ -0,0 +1,247 @@
+---
+id: development
+title: Development Best Practices
+sidebar_label: Development Best Practices
+slug: /project/development-guides
+custom_edit_url: null
+---
+
+## Defining the name of your usecase
+
+The name of the usecase must be something that refers to the business, making its real purpose clear to the business team.
+
+
+
+**Good descriptions:**
+
+✅ Search one or several funds
+
+✅ Filter users by document
+
+✅ Update unit price
+
+✅ Find the funds in Bank
+
+
+
+**Bad descriptions:**
+
+❌ list the information
+
+❌ Calculates the percentage of the fund
+
+❌ calculates the total
+
+❌ Save user in bank
+
+❌ Update Client
+
+❌ Starts the services and returns the JobId
+
+
+Reference: [https://herbsjs.org/docs/usecase/features/#creating-a-use-case](https://herbsjs.org/docs/usecase/features/#creating-a-use-case)
+
+## **Creating custom methods in the repository**
+
+**Working with PostgreSQL**
```

Comment:
This organization (Working with PostgreSQL / Don't: / Correct) is awkward. I would suggest just Good/Bad or Do/Don't.